lab1.js

```diff
@@ -77,12 +80,32 @@ var sentence2 = 'Come over here so you can scratch my belly.';
 // TODO: part #1: use a for loop to replace the words in sentence 1 with
 // 'chirp' (10 points)
 
-// your code goes here
+function chirpConverter(sentence) {
+  var newSentence = sentence.split(' ');
 
+  for (var i = 0; i < newSentence.length; i++) {
+    newSentence[i] = 'chirp';
+  }
+  return newSentence.join(', ');
```

Comment:
I think this comma is making your asserts fail. Carefully compare your console logs of sentence1 and sentence2 to the strings that are in the asserts.